c2cgeoportal/scripts/c2ctool.py

```diff
@@ -126,6 +126,17 @@ def _fill_arguments(command):
         help="Specify the remote branch",
         default="origin",
     )
+    parser.add_argument(
+        "--index-url",
+        help="An alternate camptocamp python package index URL",
+        default="http://pypi.camptocamp.net/internal-pypi/index/c2cgeoportal"
+    )
+    parser.add_argument(
+        "--c2cgeoportal-url",
+        help="An alternate c2cgeoportal egg/wheel URL (with arguments %(package) %(version)s)",
+        default="http://pypi.camptocamp.net/internal-pypi/index/%(package)s-%(version)s.tar.gz"
+    )
```

Comment:
Oh I see from https://github.com/camptocamp/c2cgeoportal/pull/1648 that we store c2cgeoportal wheels in the gh-pages branch. Are we sure that's a good idea? Each c2cgeportal wheel is about 85 MB, so won't the Git repo quickly become huge because of this?